src/components/containers/notification/Notification.js

```diff
@@ -110,29 +104,15 @@ function renderExtraAlert(alert) {
  * inverse that kind of goes against normal thinking
  */
 const CallsToAction = styled.div`
-  align-self: flex-end;
+  position: absolute;
+  bottom: -50px;
+  right: 0;
   display: flex;
   flex-direction: row-reverse;
-  padding: 0 15px 15px 0;
 
   & > button:not(:first-of-type) {
     margin-right: 15px;
   }
-
-  @media (max-width: 767px) {
-    display: block;
-    padding: 15px;
-
-    & > button {
-      display: block;
-      margin-bottom: 15px;
-      width: 100%;
-
-      &:active {
-        margin-bottom: 15px;
```

Comment:
At a glance the calls to action don't seem to be stacking when switching to mobile in the chrome dev tools. It might be the `position: absolute` causing problems with the flex box.

Reply:
Good catch.